### PR TITLE
[5.7] filesystem changes

### PR DESCRIPTION
**Pull request by Smolevich**
Related with [pull request](https://github.com/laravel/framework/pull/24452)

### CODE REVIEW

**Comment by Smolevich** [comment]
@GrahamCampbell, check this PR

File: src/Illuminate/Notifications/composer.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
         "illuminate/filesystem": "5.7.*",
         "illuminate/mail": "5.7.*",
         "illuminate/queue": "5.7.*",
-        "illuminate/support": "5.7.*"
+        "illuminate/support": "5.7.*",
+        "ramsey/uuid": "^3.0"
```
Comment (GrahamCampbell) [inline src/Illuminate/Notifications/composer.json]: ?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/Illuminate/Notifications/composer.json (hunk 1 above)
Comment (Smolevich) [reply src/Illuminate/Notifications/composer.json]: i fix it

**Comment by tillkruss** [comment]
No tests, no description, no descriptive title.

**Pull request closed without merging**